shell/platform/embedder/embedder.h

```diff
@@ -432,6 +451,11 @@ typedef struct {
   /// `FlutterPresentInfo` struct that the embedder can use to release any
   /// resources. The return value indicates success of the present call.
   BoolPresentInfoCallback present_with_info;
+  /// Specifies the initial pixel geometry of the display.
+  ///
+  /// If or when the geometry changes, use the SetPixelGeometry method to update
+  /// the engine.
+  FlutterPixelGeometry initial_pixel_geometry;
```

Comment:
This shouldn't be necessary as the embedders can be instructed to call FlutterEngineSetPixelGeometry right after a successful call to `FlutterEngineRun...`

Reply:
Ok, I'll just remove this and default it to `kUnknown` on initialization.